packages/state/recoil/selectors/stargaze.ts

```diff
@@ -0,0 +1,44 @@
+import { selectorFamily } from 'recoil'
+
+import { ChainId, WithChainId } from '@dao-dao/types'
+
+import {
+  stargazeIndexerClient,
+  stargazeWalletTotalValueUsdQuery,
+} from '../../graphql'
+
+export const stargazeWalletUsdValueSelector = selectorFamily<
+  number,
+  WithChainId<{ address: string }>
+>({
+  key: 'commonNftStargazeWalletUsdValue',
+  get:
+    ({ chainId, address }) =>
+    async () => {
+      if (
+        chainId !== ChainId.StargazeMainnet &&
+        chainId !== ChainId.StargazeTestnet
+      ) {
+        throw new Error('Expected Stargaze mainnet chain')
+      }
+
+      const { error, data } = await stargazeIndexerClient.query({
+        query: stargazeWalletTotalValueUsdQuery,
+        variables: {
+          address,
+        },
+      })
+
+      if (error) {
+        throw error
+      }
+
+      if (
+        data.wallet?.stats?.totalValueUsd === null ||
+        data.wallet?.stats?.totalValueUsd === undefined
+      )
+        throw new Error('Unexpected response from Stargaze indexer')
```

Comment:
it's possible stargaze will error like this if the wallet hasn't been seen before, so i think it would be safer to return 0 if the data seems malformed. also `data` may be undefined, so this should probably check for that too

```suggestion
      if (!data?.wallet?.stats?.totalValueUsd) {
        return 0
      }
```